src/components/EditProfile/EditTitle/index.js

```diff
@@ -1,17 +1,30 @@
 import React, { memo } from 'react'
 import propTypes from 'prop-types'
+import ButtonContainer from '../../ButtonContainer'
 import './styles.scss'
 
 const EditTitle = props => {
-  const { text } = props
+  const { text, history } = props
+
+  const closeEditPage = () => {
+    history.goBack()
+  }
+
   return (
     <div className="edit-header-container">
       <h1 className="edit-header-text">{text}</h1>
+      <ButtonContainer
```

Comment:
I did this because I was using the same button in many places and wanted a quick way to add it with the size, border-radius etc remaining the same. Although I guess you could do this equally well with an icon and a class added..